src/main/java/pl/cyber/trainees/SimpleApp.java

```diff
@@ -1,23 +1,216 @@
+    // ale z reguły powinna być to tablica args (tj. String[] args)
+
+
     public static void main(String[] args) {
-//
+
+
+//nie tworzymny metody w metodzie, powinny byc za nawiasem sklamrowym np
+        //public String nowametoda(){}
+// w ramach klasy uruchomieniowej  możemy deklarować zmienne oraz pisać kod/
+// dobrze aby metoda uruchomieniowa wywoływawała konkretne klasy z kodem który chcemy uruchomić
+
+// Należy pamietac, że metody funkcjonalne deklarujemy w taki sposób:
+        // - nie mają słówka static
+        //- modyfikator dostepu może byc public/private / protected / default
+        //Typ zwracajych informacji, aczkolwiem może metoda nic nie zwracać.
+        //void - jeśli metoda nic nie zwraca
+        //String/Integer/BigDecimal/Model/........ jeśli chcemy aby nasza metoda zwracała
+        //konkretny typ danych i była możliwa bo użycia w innym miejscu np. klasy lub projektu
+        //nazwy metod pisane camelCase'm z małej początek
+
+        System.out.println("_________________Ćwiczenie nr 1______________");
+
+
+        //pierwsze rozwiązanie
+        Basics basics = new Basics("1");
+        basics.wyswietlTekst1("tekst do wyświetlenia");
+
+        //drugie rozwiązanie
+        Basics basics2 = new Basics(" Nowy tekst do wyświetlenia ");
+        basics2.wyswietlTekst2(" tekst do wyświetlenia 2");
+
+
+        //trzecie rozwiązanie
+        Basics basics3 = new Basics("tekst1", "tekst2", " tekst3");
+        basics3.wyswietlTekst3("Nowy tekst do wyświetlenia3");
+
+        //Cwiczenie DRUGIE _INTEGER
+
+
+        BasicsInteger basicsInteger2 = new BasicsInteger();
+        basicsInteger2.wyswietlLiczbe(12343);
+        basicsInteger2.wyswietlLiczbe2(1234);
+
+        Basics testKonstr = new Basics(123);
+
+        System.out.println("_________________Ćwiczenie nr 2______________");
+
+// dodawanie liczb z konstruktora
+        BasicsInteger basicsInteger3 = new BasicsInteger("Dodajemy liczby ", 1, 2);
+
+        // M#2
+        BasicsInteger basicsInteger = new BasicsInteger();
+        String txt1 = "wynik:";
+        Integer a = 12;
+        Integer b = 12;
+
+        System.out.println("_________________Ćwiczenie nr 3______________");
+
+        basicsInteger.wyswietlanieSumyLiczb("Suma", 2, 3);
+        System.out.println(txt1);
+        System.out.println(a + b);
+
+        System.out.println("_________________Ćwiczenie nr 4 ______________\n" +
+                "Instrukcje warunkowe");
+
+        InstrukcjeWarunkowe instrukcjeWarunkowe = new InstrukcjeWarunkowe(5);
+        // instrukcjeWarunkowe.sprawdzanieRownosci(number1);
+
+        System.out.println("\n_________________Praca domowa ______________\n" +
+                "Imię i nazwisko");
+        ImieIWiek imieIWiek=new ImieIWiek();
+        imieIWiek.wczytajImie("Karolina");
+
+        ImieIWiek imieIWiek1 = new ImieIWiek();
+        Integer wiek = 28;
+        System.out.println("Twoj wiek"+SEPARACJA_SPACJA+wiek);
```

Comment:
Patrząc na kod jeśli będzie poprawny konstruktor dla klasy ImieIWiek to wtedy wystarczy 
ImieIWiek imieIWiek1 = new ImieIWiek(28);
        System.out.println("Twoj wiek"+SEPARACJA_SPACJA+imieIWiek1.getWiek());
        
        po napisanu (gettera) czyli metody zwracającej wartość wiek.

Reply:
Dziękuję !
A tak, na szczęście działa. Przyznam się że chciałam taką metodą, dla przećwiczenia i utrwalania metod.